### PR TITLE
[doc] Use relative links in docs

### PR DESCRIPTION
**Pull request by nisargthakkar**
## Use relative links in docs

Replace URL links with Markdown relative links. Github has jekyll-relative-links plugin enabled by default - [reference](https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/about-github-pages-and-jekyll#plugins). 

This helps the reader travel between docs on the same platform where they are consuming the docs from - markdown or via the docs webpage.

## How was this PR tested?

Verified locally and deployed the pages to https://nisargthakkar.github.io/venice

## Does this PR introduce any user-facing changes?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Make sure to explain your proposed changes and call out the behavior change.

### CODE REVIEW

File: docs/ops_guide/repush.md [hunk 1]
```diff
@@ -2,7 +2,6 @@
 layout: default
 title: Repush
 parent: Operator Guides
-permalink: docs/ops_guide/repush
```
Comment (FelixGV) [inline docs/ops_guide/repush.md:5]: Should we (could we?) also keep the permalinks? They look a little bit nicer. If I shared a permalink (e.g., [here](https://twitter.com/felixgv/status/1576981114866270208?s=21)) then it'll be broken after this change, right? Not a big deal, but just wondering.

**Review by FelixGV — APPROVED** [review]
Thanks! Left a minor question. I can review again if you decide to make tweaks.

File: docs/ops_guide/repush.md (hunk 1 above)
Comment (nisargthakkar) [reply docs/ops_guide/repush.md:5]: Yeah, that's true. The link will break in its current form. We can keep the permalinks if we standardize on a format

Comment (nisargthakkar) [reply docs/ops_guide/repush.md:5]: I keep going back and forth in my head b/w having permalinks or not. I think I agree that having permalinks is a better idea. Updated the permalinks

**Review by sixpluszero — APPROVED** [review]

**Pull request merged**